### PR TITLE
Add new version of pipeline v0.1.1

### PR DESCRIPTION
**Pull request by biblesyme**
For more detailed version information, please refer to https://github.com/rancher/pipeline/releases/tag/v0.1.1

Related issue is [here](https://github.com/rancher/rancher/issues/10772)

### CODE REVIEW

**Comment by deniseschannon** [comment]
since this template has the same `minimum_rancher_version` as template, should we hide the original template for fresh installs? this can be done by adding `maximum_rancher_version: v1.6.13-rc1`? Also, did you test upgrade from one template to this new one?

**Comment by biblesyme** [comment]
@deniseschannon Updated and Testd it in upgrading v0.1.0 to v0.1.1.

File: infra-templates/pipeline/0/rancher-compose.yml [hunk 1]
```diff
@@ -3,6 +3,7 @@ catalog:
   name: Pipeline
   version: v0.1.0
   minimum_rancher_version: 1.6.13-rc2
+  maximum_rancher_version: 1.6.13-rc1
```
Comment (deniseschannon) [inline infra-templates/pipeline/0/rancher-compose.yml]: let's have this match whatever the `min` version is. I didn't notice that it was set to rc2 before.

File: infra-templates/pipeline/1/docker-compose.yml.tpl [hunk 2]
```diff
@@ -1,8 +1,8 @@
 {{- $jenkinsMasterImage:="jenkins/jenkins:2.60.2-alpine"}}
-{{- $jenkinsBootImage:="rancher/jenkins-boot:v0.1.1"}}
-{{- $jenkinsSlaveImage:="rancher/pipeline-jenkins-slave:v0.1.0"}}
-{{- $pipelineServerImage:="rancher/pipeline:v0.1.0"}}
-{{- $pipelineUIImage:="rancher/pipeline-ui:v0.1.0"}}
+{{- $jenkinsBootImage:="rancher/jenkins-boot:v1.0.0"}}
```
Comment (deniseschannon) [inline infra-templates/pipeline/1/docker-compose.yml.tpl]: we renamed this to `rancher/pipeline-jenkins-boot:v1.0.0`

**Comment by deniseschannon** [comment]
LGTM

**Pull request merged**